### DISCUSSION TITLE
Blacklist (or whitelist) certain apps?

### DISCUSSION THREAD

**Discussion by TheRoyalScam** [noah-nuebling/mac-mouse-fix#843] [Questions]
I’ve got a couple apps that get real finicky with smooth scrolling enabled (Pro Tools, XnConvert). Mos allows me to block those specific apps from smooth scrolling, reverse scrolling or both. 

Any similar ability in MMF 3? If not, could it be added?

**Comment by kelegele** [comment]
I need this 🧎‍♂️🧎‍♂️🧎‍♂️
